### ISSUE TITLE
Update produces log: "Failed to execute operation: File exists"

### ISSUE THREAD

**Issue by 2xB** [influxdata/telegraf#15392]
### Relevant telegraf.conf

```toml
None
```

### Logs from Telegraf

```text
Updating   : telegraf-1.30.3-1.x86_64                                                                             2/4
warning: /etc/telegraf/telegraf.conf created as /etc/telegraf/telegraf.conf.rpmnew
Failed to execute operation: File exists
```

### System info

telegraf-1.30.3-1.x86_64, CentOS7

### Steps to reproduce

Updating an (old) `telegraf-1.30.2-1.x86_64` installation to `telegraf-1.30.3-1.x86_64` in CentOS 7 using `yum update`.

### Expected behavior

No error

### Actual behavior

Above log output

**Comment by powersj** [comment]
Hi,

The rpmnew file is created when a config file was modified from the original state. The RPM install will not overwrite your changes and creates the new file with the .rpmnew extension.

In your case it looks like you have probably updated previously, so you already have an .rpmnew file. So on the second update, when rpm/dnf/yum/etc. tries to create the file again it conflicts.

I am not sure there is anything Telegraf can or should do about this. This is your package manager giving you a warning that you are about to overwrite a previous rpmnew file.

**Comment by 2xB** [comment]
I do not know if `Failed to execute operation: File exists` correlates with the warning message above. I got the message during updating GitLab as well, but there did not get any warning beforehand. If the warning in case of Telegraf comes from the package manager, I therefore find it rather likely that these are independent log entries.

**Comment by powersj** [comment]
> I therefore find it rather likely that these are independent log entries.

Independent of what? These log messages are all from your package manager.

**Comment by 2xB** [comment]
I think the warning and the `Failed to execute operation: File exists` message come from different files, because later in the update log - during a GitLab upgrade -, I got another `Failed to execute operation: File exists` without a prior warning.

My assumption was naively that if it was a problem with my package manager, it would be more verbose, tell me what exactly failed and highlight problems in the post-transaction summary. Since I did not get any errors reported at the end of running `yum install` and there were not any details, this being the package manager seemed unlikely.

Thank you very much, in that case I just ignore this message for now
